ui/packages/consul-ui/app/templates/dc/acls/policies/-form.hbs

```diff
@@ -28,53 +28,86 @@
     {{/if}}
 {{/if}}
     <div>
-{{#if (and create (can "create tokens")) }}
+      <Hds::ButtonSet>
+        {{#if (and create (can "create tokens")) }}
         {{! we only need to check for an empty name here as ember munges autofocus, once we have autofocus back revisit this}}
-        <button type="submit" {{ action "create" item}} disabled={{if (or item.isPristine item.isInvalid (eq item.Name '')) 'disabled'}}>Save</button>
-{{ else }}
-  {{#if (can "write policy" item=item)}}
-        <button
-          type="submit" {{ action "update" item}} disabled={{if item.isInvalid 'disabled'}}>Save</button>
-  {{/if}}
-{{/if}}
-        <button type="reset" {{ action "cancel" item}}>Cancel</button>
-{{# if (and (not create) (can "delete policy" item=item) ) }}
-        <ConfirmationDialog @message="Are you sure you want to delete this Policy?">
+          <Hds::Button
+            @text='Save'
+            type='submit'
+            {{ action 'create' item}}
```

Comment:
I think you'll have to do the same thing we talked about for roles here. It's also using the same mixin.

Reply:
updated where found disabled + {{action}} on a button